iib/common/tracing.py

```diff
@@ -0,0 +1,211 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""Configures the Global Tracer Provider and exports the traces to the OpenTelemetry Collector.
+
+The OpenTelemetry Collector is configured to receive traces via OTLP over HTTP.
+The OTLP exporter is configured to use the environment variables
+OTEL_EXPORTER_OTLP_TRACES_ENDPOINT and OTEL_EXPORTER_OTLP_TRACES_HEADERS to
```

Comment:
Is this still true as I see that `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` has been replaced by ` endpoint=f"{os.getenv('OTEL_EXPORTER_OTLP_ENDPOINT')}/v1/traces",` https://github.com/release-engineering/iib/blob/d9a9f502e7b8ec0ac9115e3f4969337caaa35cf0/iib/common/tracing.py#L61 

could you please modify the comment in this file or in the README file where you added a description about the var to explain this change?

Reply:
yUp. Tweaked it, take alook and let me know please?